File: Topic0102/012_sum_the_evens/sum_the_evens.cpp
#include <iostream>

bool isEven(int);
int sec(int);

int main(){
  int x = 3;

  std::cout << x << " : " << sec(x) << std::endl;

  return 0;
}

bool isEven(int n){
  if(n % 2 == 0) return true;
  return false;
}

int sec(int n){
  int y = 2;
  
  for(int i = 1; i <= n; i++)
    if(isEven(i)) y += i;

  return y;
}

File: Topic0102/012_sum_the_evens/sum_the_evens.py
def isEven(n):
    if n % 2 == 0:
        return True
    return False

def sec(n):
    y = 2
    for i in range(1, n+1):
        if isEven(i):
            y += i
    return y

x = 3

print(f'{x} : {sec(x)}')
